keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -357,13 +393,15 @@ generated is valid OpenAPI v3.
 #### Beta
 
 - Native types are updated to capture capabilities introduced with v3
+  - anyOf is correctly used for types that can take on the value of multiple types.
```

Comment:
To be precise this is about two cases:
1. intOrString is typed incorrectly today (as string? but definitely not polymorphic with anyOf)
2. other types that can benefit from anyOf to get a more complete type and which are just `type: ""` today

(1) is a blocker for beta. (2) is cosmetics, but good to have.

Reply:
Thanks for the clarification. I'll keep just 1 as part of the beta requirements. It's most likely that 2 will also be done for beta but we can keep that outside the blocking tasks in the KEP.